### PR TITLE
main: Fix CLI quickstarts

### PR DESCRIPTION
**Pull request by geky**
See https://github.com/veracruz-project/veracruz/pull/309 for more info

I've cherry-picked this commits onto main, but am not in a position to test on Nitro at the moment.

I think these should eventually be moved to Linux and added to CI, but in the meantime this PR gets them into a better state.

### CODE REVIEW

**Review by dominic-mulligan-arm — APPROVED** [review]
LGTM, though will wait for a second opinion from @ShaleXIONG before merging.

**Review by ShaleXIONG — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
+1+1 = +1 reached, merging.

**Pull request merged**
